File: manager/src/main/java/com/mydemo/manager/service/UserService.java
package com.mydemo.manager.service;

import com.mydemo.manager.entity.JzUser;

import java.util.List;

public interface UserService {
    List<JzUser> findAll();
    JzUser findOne(Integer id);
    int setStatus(Integer[] ids,String sta);
    List<JzUser> findBySta(String sta);
}

File: manager/src/main/resources/templates/js/service/userService.js
app.service('userService',function($http){
    this.findOne=function (id) {
        return $http.get('../user/findOne?id='+id);
    }
    this.findAll=function () {
        return $http.get('../user/findAll');
    }
    this.setStatus=function (ids,sta) {
        return $http.get('../user/setStatus?ids='+ids+'&sta='+sta);
    }
    this.findBySta=function (sta) {
        return $http.get('../user/findBySta?sta='+sta)
    }
})